### DISCUSSION TITLE
Is there no method for retrieving the price of a purchased product?

### DISCUSSION THREAD

**Discussion by timyc** [imdhemy/laravel-in-app-purchases#125] [Q&A] [answered]
From the ProductPurchase class, I don't see any method for getting the price of the purchased item. After a successful purchase, I would like to verify that the gross amount is equal to the value I have stored in my database.

**Comment by imdhemy** [comment, marked as answer]
The `ProductPurchase` class represents the [Google Play resource](https://developers.google.com/android-publisher/api-ref/rest/v3/purchases.products#resource:-productpurchase) with the same name. Which unfortunately does not
 have price attribute, unlike the `SubscriptionPurchase` class which has price and currency attributes.
